### ISSUE TITLE
Laser USA Power Plant gives more power but looks identical to regular USA Power Plant

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#861]
Laser USA Power Plant gives more power but looks identical to regular USA Power Plant.

USA has green rods.

Superweapon General has blue rods.

Laser General currently has green rods, but can have purple rods.

https://www.moddb.com/mods/nproject-mod/images/advanced-cold-fusion-reactor

**Comment by xezon** [comment]
commy2
> SWG produces 5 only. The difference is the Control Rods upgrade. So any visual difference should exist before Control Rods.

**Comment by ZekeDlyoung** [comment]
> SWG produces 5 only. The difference is the Control Rods upgrade. So any visual difference should exist before Control Rods.

True, but this is a simple change which we already have the assets for 🙃 

I do have this, but it has a somewhat larger footprint than the normal reactor:
https://www.moddb.com/mods/generation-x/images/advanced-cold-fusion-reactor

**Comment by commy2** [comment]
Advanced Reactor from Nuke General also looks slightly different and has a different portrait. It produces 18 power instead of 10 of vChina/Inf/Tank.

**Comment by xezon** [comment]
Maybe we can have Laser General Power Plant start with the green rods (maybe just partly), and then upgrade to the purple rods?

**Comment by ZekeDlyoung** [comment]
> Maybe we can have Laser General Power Plant start with the green rods (maybe just partly), and then upgrade to the purple rods?

That could work too, but some people might see the already exposed rods and assume that it's a bug.

The purple rods are already known by people who play mods, and people who are curious about cut ZH content in general (kinda like the black and white Aurora models), so it's a bit more intuitive in a sense

**Comment by xezon** [comment]
Maybe the green rods can be just on half height before Upgrade, and then extend fully with purple after Upgrade.

**Comment by ZekeDlyoung** [comment]
Seems a bit too complicated.

IMO the purple rods still make sense. You can say that SupW Gen's reactor uses the same core as the standard USA CFR, but it interacts differently with the Advanced Control Rods to generate more power. Meanwhile, Townes' reactor uses both a different core and different control rods to increase its output even before the upgrade

**Comment by ExiLe-ZH** [comment]
Usa always looked like yellow to me, not green.

But yea, red would be nice for laser imo, since his laser pats/tanks also shoot red beams.

**Comment by xezon** [comment]
Red rods sounds nice.

**Comment by MTKing4** [comment]
I also like it to be red

Also since this is expanding from one issue to another, should Particle cannon have Red beam for Laser? 😄 
They did intend to do that i think, there's an unused Laser Cannon building that works in wb, same model but different name, and it doesn't have a different portrait.

**Comment by commy2** [comment]
> should Particle cannon have Red beam for Laser?

No reason to change that in a patch, but we could fix up the Laser Cannon building to have that and at least add localization, so the super weapon timer doesn't say MISSING blah.

**Comment by MTKing4** [comment]
> > should Particle cannon have Red beam for Laser?
> 
> No reason to change that in a patch, but we could fix up the Laser Cannon building to have that and at least add localization, so the super weapon timer doesn't say MISSING blah.

Yeah that would be nice actually, to at least have that there, would be harmless
